ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1146,17 +1327,22 @@ namespace OrdinalAxis {
                     // range, else it happens on the current x axis which is
                     // smaller and faster.
                     chart.fixedRange = max - min;
+
+                    // @todo debug session, to change later
```

Comment:
> to change later

When? We should remove debug stuff before merging it into the master

Reply:
My bad, forgot to delete it before pushing.